B827EBFFFE831835.json

```diff
@@ -1,19 +0,0 @@
-{
-        "gateway_conf": {
-                "gateway_ID": "B827EBFFFE831835",
-                "servers": [ 
-                     { 
-                    "server_address": "router.us-west.thethings.network", 
```

Comment:
Can you change "router.us-west.thethings.network" to "router.us.thethings.network"
Thank you